internals-js/src/__tests__/definitions.test.ts

```diff
@@ -427,6 +427,10 @@ test('handling of descriptions', () => {
       pages: Int @Important
     }
 
+    extend type Book {
```

Comment:
do we need to check the result anywhere?

Reply:
The test ends by comparing the result of `printSchema` and the original input. So before this patch the test would fail due the bug solved here.